doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -167,133 +165,74 @@ Enable partition mode in ACRN hypervisor
+		.pbdf.bits = {.b = 0x00U, .d = 0x15U, .f = 0x00U},
+	},
+	{
+		.vbdf.bits = {.b = 0x00U, .d = 0x02U, .f = 0x00U},
+		.pbdf.bits = {.b = 0x02U, .d = 0x00U, .f = 0x00U},
+	},
+     };
+     ...
+
+#. Optionally, configure the ``VMx_CONFIG_OS_BOOTARGS`` kernel command line arguments
+
+   The partiton mode configure infomation is located in header
+   file ``hypervisor/arch/x86/configs/apl-mrb/partition_config.h``.
    The kernel command line arguments used to boot the privileged VMs are
    hardcoded as ``/dev/sda3`` to meet the Clear Linux OS automatic installation.
    In case you plan to use your customized root
    filesystem, you may optionally edit the ``root=`` parameter specified
-   in the ``.bootargs`` field of the ``.vm_desc_array`` structure, to
-   instruct the Linux kernel to mount the right disk partition:
+   in the ``VMx_CONFIG_OS_BOOTARGS`` MACRO, to instruct the Linux kernel to
+   mount the right disk partition:
    
    .. code-block:: none
-     :emphasize-lines: 9-11,20-22
-     :caption: hypervisor/partition/vm_description.c
+     :emphasize-lines: 12-14
+     :caption: hypervisor/arch/x86/configs/apl-mrb/partition_config.h
 
      ...
-     /* Virtual Machine descriptions */
-     .vm_desc_array = {
-        {
-           /* Internal variable, MUSTBE init to -1 */
-           .vm_hw_num_cores = VM1_NUM_CPUS,
-           .vm_pcpu_ids = &VM1_CPUS[0],
-           ...
-           .bootargs = "root=/dev/sda3 rw rootwait noxsave maxcpus=2 nohpet console=hvc0 \
-                        console=ttyS2 no_timer_check ignore_loglevel log_buf_len=16M \
-                        consoleblank=0 tsc=reliable xapic_phys",
-           .vpci_vdev_array = &vpci_vdev_array1,
-           .mptable = &mptable_vm1,
-        },
-        {
-           /* Internal variable, MUSTBE init to -1 */
-           .vm_hw_num_cores = VM2_NUM_CPUS,
-           .vm_pcpu_ids = &VM2_CPUS[0],
-           ...
-           .bootargs = "root=/dev/sda3 rw rootwait noxsave maxcpus=2 nohpet console=hvc0 \
-                        console=ttyS2 no_timer_check ignore_loglevel log_buf_len=16M \
-                        consoleblank=0 tsc=reliable xapic_phys",
-           .vpci_vdev_array = &vpci_vdev_array2,
-           .mptable = &mptable_vm2,
-           .lapic_pt = true,
-        },
-     }
+     #define	VM0_CONFIGURED
+
+     #define VM0_CONFIG_NAME			"PRE-LAUNCHED VM1 for APL-MRB"
+     #define VM0_CONFIG_TYPE			PRE_LAUNCHED_VM
+     #define VM0_CONFIG_PCPU_BITMAP		(PLUG_CPU(0) | PLUG_CPU(2))
+     #define VM0_CONFIG_FLAGS			IO_COMPLETION_POLLING
+     #define VM0_CONFIG_MEM_START_HPA		0x100000000UL
+     #define VM0_CONFIG_MEM_SIZE		0x20000000UL
+
+     #define VM0_CONFIG_OS_NAME			"ClearLinux 26600"
+     #define VM0_CONFIG_OS_BOOTARGS		"root=/dev/sda3 rw rootwait noxsave maxcpus=2 nohpet console=hvc0 \
+						console=ttyS2 no_timer_check ignore_loglevel log_buf_len=16M \
+						consoleblank=0 tsc=reliable xapic_phys"
+
```

Comment:
I would recommend to add a note here to indicate the the root device for `VM1` also is `/dev/sda3` since the USB controller is the only one seen in that VM.